src/applicationManager.ts

```diff
@@ -513,9 +516,85 @@ export async function setResourceAccess({
       },
     );
     await errorHandler("Granting graph api permissions", assignRolesResult);
+  } else {
+    console.log("No graph api permissions to grant");
+  }
+}
+
+function _buildResourceAccess(roleIds: string[], accessType: string) {
```

Comment:
no need to prefix with `_` if you don't export the function it won't be available

Reply:
prefix with `_` purely for readability. I know it is a private function when seeing it 😃